.github/actions/setup-project/action.yml

```diff
@@ -0,0 +1,32 @@
+name: Setup Project
+
+inputs:
+  pnpm-version:
+    required: false
+    default: 8.0.0
+  node-version:
+    required: false
+    default: 18
+  turbo-version:
+    required: false
+    default: 1.10.7
+
+runs:
+  using: composite
+  steps:
+    - uses: pnpm/action-setup@v2
+      with:
+        version: ${{ inputs.pnpm-version }}
+    - uses: actions/setup-node@v3
+      with:
+        node-version: ${{ inputs.node-version }}
+        cache: "pnpm"
+    - name: Install Turbo
+      run: npm i -g turbo@${{ inputs.turbo-version }}
+      shell: bash
+    - name: Install dependencies
+      run: pnpm install
+      shell: bash
```

Comment:
Do we need to specify that is bash for each step?

Reply:
Yes, it's required in composite actions

https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsstepsshell

Reply:
We can choose a different shell if we need, but bash should do for now

Reply:
Thank you for the clarification.